projects/addon-mobile/components/sheet/components/sheet/sheet.providers.ts

```diff
@@ -18,9 +18,9 @@ export const TUI_SHEET_PROVIDERS: Provider[] = [
         useFactory: ({nativeElement}: ElementRef<HTMLElement>): Observable<boolean> => {
             return merge(
                 tuiTypedFromEvent(nativeElement, `touchstart`, {passive: true}).pipe(
-                    mapTo(true),
+                    map(() => true),
```

Comment:
```suggestion
                    map(ALWAYS_TRUE_HANDLER),
```

https://github.com/Tinkoff/taiga-ui/blob/4b799b5f5fb98eeda678b5aa7af82f9901544667/projects/cdk/constants/always-true-handler.ts#L4

Reply:
fixed